histomicstk/saliency/tests/tissue_detection_test.py

```diff
@@ -105,3 +105,5 @@ def suite():
 if __name__ == '__main__':
     runner = unittest.TextTestRunner(failfast=True)
     runner.run(suite())
+    # cleanup
+    shutil.rmtree(savepath)
```

Comment:
This is also removed in `test_get_tissue_boundary_annotation_documents`.  Either each test should create and remove a directory, or this should be changed.

Would it be better to have this in a try: finally: block to ensure the test directory gets removed?